aea/configurations/base.py

```diff
@@ -994,6 +994,7 @@ def __init__(
         aea_version: str = "",
         fingerprint: Optional[Dict[str, str]] = None,
         fingerprint_ignore_patterns: Optional[Sequence[str]] = None,
+        entrypoint: Optional[str] = None,
```

Comment:
Probably should be more explicitly labelled `build_entrypoint` as there might be other entry points one day..